package.json

```diff
@@ -130,6 +131,7 @@
     "lodash": "^4.17.21",
     "npm-check-updates": "^16.1.0",
     "nx": "^14.8.3",
+    "nyc": "^15.1.0",
```

Comment:
new dependency, we have a process for this; I would talk to Danielle about this, or remove if not necessary

Reply:
Removed the dependency